scan-payment/src/main/java/com/getbouncer/scan/payment/ml/SSDObjectDetect.kt

```diff
@@ -98,6 +98,24 @@ fun RectF.scaled(scaledSize: Size): RectF {
     )
 }
 
+fun RectF.centerScaled(width: Float, height: Float): RectF {
+    return RectF(
+        this.centerX() - this.width() * width / 2,
+        this.centerY() - this.height() * height / 2,
+        this.centerX() + this.width() * width / 2,
+        this.centerY() + this.height() * height / 2
+    )
+}
```

Comment:
Currently, `DebugOverlay` has a method `RectF.scaled(size)`. Can we relocate that method as well as this method to the `Layout.kt` class? Additionally, can you rename `width` and `height` to `scaleX` and `scaleY` respectively, since they don't really represent the new width or height?